main.go

```diff
@@ -6,14 +6,15 @@ import (
 	"net/http"
 
 	"github.com/gorilla/mux"
-	"gobootcamp.com/controllers"
+	controllers "gobootcamp.com/controller"
 )
 
 func main() {
 	router := mux.NewRouter()
 
 	router.Path("/pokemons").HandlerFunc(controllers.GetPokemons).Methods("GET")
```

Comment:
if you can, you can decouple the definition endpoints into a new layer called router :)